servicetalk-http-api/src/main/java/io/servicetalk/http/api/Service.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright © 2022 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.servicetalk.http.api;
+
+/**
+ * Parent interface for all available HTTP services.
+ *
+ * @see HttpService
+ * @see StreamingHttpService
+ * @see BlockingHttpService
+ * @see BlockingStreamingHttpService
+ */
+public interface Service extends HttpExecutionStrategyInfluencer {
```

Comment:
As we discussed offline, let's consider an http-specific name, like `HttpServiceBase` or `BaseHttpService`.